ibl_to_nwb/brainwide_map/read_after_write_testing.py

```diff
@@ -0,0 +1,44 @@
+import os
+import numpy as np
+import pandas as pd
+from pathlib import Path
+import h5py
+from pynwb import NWBHDF5IO, NWBFile
+from one.api import ONE
+import logging
+import read_after_write as raw
+
+# the session
+eid = "caa5dddc-9290-4e27-9f5e-575ba3598614"  # dual probe session
+
+# local setup
+base_path = Path.home() / "ibl_scratch"
+nwb_path = base_path / "nwbfiles" / f"{eid}" / f"{eid}.nwb"
+cache_folder = base_path / "ibl_conversion" / eid / "cache"
+
+# ONE instantiation
+os.makedirs(cache_folder, exist_ok=True)
+
+one = ONE(
+    base_url="https://openalyx.internationalbrainlab.org",
+    password="international",
+    silent=False,
+    cache_dir=cache_folder,
+)
+
+# one_revision = dict(revision='2024-05-06')
+
+# NWB file
+h5py_file = h5py.File(nwb_path, "r")
+io = NWBHDF5IO(file=h5py_file, load_namespaces=True)
+nwbfile = io.read()
+
+raw.test_IblSortingInterface(nwbfile, one, eid)
+raw.test_WheelInterface(nwbfile, one, eid)
+raw.test_RoiMotionEnergyInterface(nwbfile, one, eid)
+raw.test_BrainwideMapTrialsInterface(nwbfile, one, eid)
+raw.test_IblPoseEstimationInterface(nwbfile, one, eid)
+raw.test_LickInterface(nwbfile, one, eid)
+raw.test_PupilTrackingInterface(nwbfile, one, eid)
```

Comment:
The step of reading the NWB file and running the tests on it could be its own function, and we could even back it into the `run_conversion` procedure so that it's checked every time and errors out if there's ever an issue